src/js/components/Drop/__tests__/__snapshots__/Drop-test.js.snap

```diff
@@ -255,7 +259,9 @@ exports[`Drop align left right top bottom 1`] = `
 `;
 
 exports[`Drop align left right top bottom 2`] = `
-".iVWgBr {
+"
+
```

Comment:
i think you are running and old version of styled components. you need to delete your node modules and run npm install again